modules/migration-mtc-workflow.adoc

```diff
@@ -6,7 +6,7 @@
 [id="migration-mtc-workflow_{context}"]
 = {mtc-full} workflow
 
-You use the {mtc-full} ({mtc-short}) to migrate Kubernetes resources, persistent volume data, and internal container images from to {product-title} {product-version} by using the {mtc-short} web console or the Kubernetes API.
+The {mtc-full} ({mtc-short}) helps you migrate Kubernetes resources, persistent volume data, and internal container images to {product-title} {product-version} by using the {mtc-short} web console or the Kubernetes API.
```

Comment:
> Do not assign human characteristics to inanimate objects, which are incapable of human behaviors and emotions. As much as possible, focus technical information on users and their actions, not on a product and its actions. 

Are you sure you want to make this whole change?

Reply:
For some reason, I thought we were supposed to say "x helps you to do y" because I see it frequently. I reverted the change.